shared/@hedvig-ui/Modal/index.tsx

```diff
@@ -114,6 +119,12 @@ export const Modal = (props: ModalProps) => {
 
   useClickOutside(modalRef, clickOutsideCloseHandler)
 
+  useEffect(() => {
+    if (isEscapePressed) {
+      props.onClose()
+    }
+  }, [isEscapePressed])
+
```

Comment:
Nice, almost always expected behavior 😄 

Optional, but you could also just do

```suggestion
useKeyIsPressed(Keys.Escape, () => props.onClose())
```

on line 110